src/Discord.Net.Examples/Discord.Net.Examples.csproj

```diff
@@ -15,7 +16,7 @@
   <ItemGroup>
     <ProjectReference Include="..\Discord.Net.Core\Discord.Net.Core.csproj" />
     <ProjectReference Include="..\Discord.Net.WebSocket\Discord.Net.WebSocket.csproj" />
-    <PackageReference Include="JetBrains.Annotations" Version="2019.1.3" />
+    <PackageReference Include="JetBrains.Annotations" Version="2020.1.0" />
```

Comment:
Isnlt this a good analyzer that could be enabled on all of the Discord.net projects instead of just this one?